owasp-suppressions.xml

```diff
@@ -46,6 +46,15 @@
         <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@.*$</packageUrl>
         <vulnerabilityName>CVE-2023-4586</vulnerabilityName>
     </suppress>
+    <suppress until="2023-12-31Z">
```

Comment:
Please fix rather than suppressing this one and above. Latest framework libs/grpc resolve the grpc and netty vulns.